cmd/livepeer/livepeer.go

```diff
@@ -254,14 +260,14 @@ func main() {
 
 		//Get the Eth client connection information
 		gethUrl := ""
-		if *ethIpcPath != "" {
+		if *ethUrl != "" {
 			//Connect to specified IPC file
-			gethUrl = *ethIpcPath
+			gethUrl = *ethUrl
 		} else if *ethUrl != "" {
```

Comment:
This part isn't needed anymore

Reply:
Ah yes, I had that change locally. Was going to push today!